webgpu/create_capture.ts

```diff
@@ -20,7 +20,7 @@ export interface CreateCapture {
  * Creates a texture and buffer to use as a capture.
  *
  * @example
- * ```ts, no-eval
+ * ```ts no-eval
```

Comment:
For context: https://github.com/denoland/deno_doc/issues/583

Reply:
already fixed in #4813